man/syncthing-rest-api.7

```diff
@@ -261,7 +261,7 @@ Returns the current configuration.
     \(dqoverwriteRemoteDeviceNamesOnConnect\(dq: false,
     \(dqtempIndexMinBlocks\(dq: 10,
     \(dqunackedNotificationIDs\(dq: [
-      \(dqauthenticationUserAndPassword\(dq
+      \(dqguiAuthentication\(dq
```

Comment:
Same here, generated from the docs repo.

Reply:
Fixed in 9811751861a3551791c39f4d642fce98ffe8f7c0.